site/src/examples/input/Bordered.tsx

```diff
@@ -0,0 +1,9 @@
+import { ReactElement } from "react";
+import { Input, FlowLayout } from "@salt-ds/core";
+
+export const Bordered = (): ReactElement => (
+  <FlowLayout style={{ maxWidth: "256px" }}>
+    <Input bordered defaultValue="Value" />
```

Comment:
When status is set (e.g. error)
- hover hides the activation indicator. Feels a bit weird, worth design confirmation.
- full border should turn into status color? (shown as gray right now)

Reply:
fixed this